.pre-commit-config.yaml

```
@@ -42,6 +42,25 @@ repos:
       - id: check-yaml
         name: "Check YAML"
 
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args: [--target-version=py39]
+        files: "pep_sphinx_extensions/tests/.*"
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        args: [--profile=black]
```

If we make the above change, we'll want `--line-length=79` here. Using `--atomic` might be a good idea, which avoids making breaking changes.

```suggestion
        args: ['--profile=black', '--line-length=79', '--atomic']
```

We could also use `--add_import="from __future__ import annotations"` with this if we wanted to automatically ensure newer annotation syntax will work on older Pythons...though in that case, we'd probably want to do so for the rest of the build system too.

---

We only need to support 3.9+ because we're using `startswith`. Maybe add `from __future__ import annotations` ass and when we need it?

---

Personally, I prefer to let tooling handle mechanical bookkeeping consistently rather than expending mental energy determining whether it is necessary in each and every file and manually adding it. But for now, since we only support 3.9 regardless, don't use any Py3.10+ annotations yet and the path forward on that isn't fully clear, I don't think its necessary to add that yet; I was just suggesting it as a future option if we do end up wanting/needing that.